### PR TITLE
Feature/pkg maintenance

### PR DESCRIPTION
**Pull request by cmccully**
Package maintenance. Adding Jenkinsbuild. Updated package template. Etc.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		    steps {
+		        script {
+                    sh 'docker run --rm -w=/lco/banzai/ --user=root "${DOCKER_IMG}" python setup.py test'
+		        }
+		    }
+		}
+		stage('Deploy') {
+			parallel {
+				stage('Dev') {
+					when { branch 'dev' }
```
Comment (kurczynski) [inline Jenkinsfile]: The Banzai branch seems to be named `develop`, this is looking for the `dev` branch.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM docker.lcogt.net/miniconda3:4.0.5
+FROM docker.lco.global/docker-miniconda3:4.3.21-6-ga718de9
```
[the remark this answers is not in the document]
Comment (kurczynski) [reply Dockerfile]: Change this to:
```
ARG CONDA_VER
FROM docker.lco.global/docker-miniconda3:${CONDA_VER}
```

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+		dockerImage = null
+		PROJ_NAME = projName("${JOB_NAME}")
+		GIT_DESCRIPTION = gitDescription()
+		DOCKER_IMG = dockerImageName("${LCO_DOCK_REG}", "${PROJ_NAME}", "${GIT_DESCRIPTION}")
+	}
+	stages {
+		stage('Build image') {
+			steps {
+				script {
+					dockerImage = docker.build("${DOCKER_IMG}")
```
Comment (kurczynski) [inline Jenkinsfile]: You can pass build args like so `docker.build("${DOCKER_IMG}", "--build-arg CONDA_VER=4.3.21-6-ga718de9 .")`

File: Dockerfile [hunk 4]
```diff
@@ -1,4 +1,5 @@
-FROM docker.lco.global/docker-miniconda3:4.3.21-6-ga718de9
+ARG MINICONDA_VERSION=4.4.10
```
[the remark this answers is not in the document]
Comment (cmccully) [reply Dockerfile]: Right. But is it bad to include a default value?

[the remark this answers is not in the document]
Comment (kurczynski) [reply Dockerfile]: No, but it'll probably get stale. And it forces you to make a conscious decision on the version being used.

**Pull request merged**